crates/biome_cli/tests/snapshots/main_cases_biome_json_support/check_biome_json.snap

```diff
@@ -27,7 +27,7 @@ debugger;
 # Emitted Messages
 
 ```block
-Fixed 1 file(s) in <TIME>
+Checked 1 file(s) in <TIME>; 1 file(s) are changed, 0 file(s) are unchanged.
```

Comment:
Because we write `Checked X files`, I  could write `Chnaged X files`.
Also, I could remove the parens around the `s`, that's ok to be wrong for `1` or otherwise we could add the s only when we have more than 1 file.

By the way I am not sure if we need to display both changed and unchanged. We can infer from the number of checked and changed files, the number of unchanged files.

For example:

- `Checked 1 files in <TIME>; Changed 1 files.`
  Thus, we have 0 unchanged files.
- `Checked 3 files in <TIME>; Changed 1 files.`
  Thus, we have 2 unchanged files.
- `Checked 1 files in <TIME>; Changed 0 files.`
  Thus, we have 1 unchanged files.

Reply:
Also, we could reuse the `Fixed` wording instead of `Changed`. This allows to quickly distinguish `Checked` from `Fixed`.

Reply:
Agreed with the comments thus far. I would even go one step further and apply some more special handling for the `0 files` case. `Fixed 0 files` sounds a bit unsatisfactory IMO. "Why was nothing fixed?" "Is something going wrong?" Those are not questions you want to evoke.

If no files needed to be changed, I'd say `No fixes necessary`.

`Checked 358 files in 24ms. No fixes necessary.`

That feels good :)

Reply:
Amazing feedback everyone! That's what I was looking for :)